src/views/bundler/Bundler.tsx

```diff
@@ -90,7 +99,77 @@ function Bundler(props: any) {
         setRows(rows);
         setTableLoading(false);
     };
+ 
+    // const checkTermInNetworks = async (term) => {
+    //     const networksWithTerm = [];
+    //     const networkKeys = Object.keys(NETWORK_LIST);
+      
+    //     await Promise.all(networkKeys.map(async (networkValue) => {
+    //       try {
+    //         const response = await fetch(`https://api.jiffyscan.xyz/v0/searchEntry?entry=${encodeURIComponent(term)}&network=${NETWORK_LIST[networkValue].key}`);
+    //         const data = await response.json();
+     
+           
+    //         if (!data.message ) {
+    //             console.log(data)
+    //           networksWithTerm.push(networkValue);
+    //         }
+    //       } catch (error) {
+    //         console.error(`Error fetching data for network ${networkValue}:`, error);
+    //       }
+    //     }));
+    //     console.log(networksWithTerm)
+    //    setDisplayNetworkList(networksWithTerm.map(index => NETWORK_LIST[index]));
+    //    setNetworkListReady(true)
+    //     // return networksWithTerm;
+    //    };
+    // if(!networkListReady)
+    //    checkTermInNetworks(hash)
+    const checkTermInNetworks = React.useCallback(async (term : string) => {
+        const networksWithTerm: string[] = [];
+        const networkKeys = Object.keys(NETWORK_LIST);
+    
+        try {
+          const responses = await Promise.all(
```

Comment:
The API call made is being duplicated across files. Best to have it in one area. 

https://thevaluable.dev/dry-principle-cost-benefit-example/

Reply:
common part moved to api file in - [cf8bf85](https://github.com/jiffy-labs/jiffyscan-frontend/pull/129/commits/cf8bf85bc6ecfbfc23f5300811992a8eb84a22de)